CIP-0050/README.md

```diff
@@ -1,431 +1,250 @@
 ---
 CIP: 50
 Title: Pledge Leverage-Based Staking Rewards
-Status: Proposed
 Category: Ledger
+Status: Proposed
 Authors:
   - Michael Liesenfelt <michael.liesenfelt@gmail.com>
-Implementors: N/A
+  - Ryan Wiley
+  - Rich Manderino [ECP]
+  - Stef M [RABIT]
+  - Wayne [OTG]
+  - Homer J (AAA)
+  - Chad [BBHMM]
+Implementors: []
 Discussions:
   - https://github.com/cardano-foundation/CIPs/pull/242
   - https://forum.cardano.org/t/cip-shelley-s-basho-voltaire-decentralization-update/97685
   - https://forum.cardano.org/t/minimum-pool-fees-with-a-brief-mention-of-k-changes/97002/82
-Created: 2022-04-04
-License: CC-BY-4.0
+  - https://github.com/cardano-foundation/CIPs/pull/1042/files
```

Comment:
```suggestion
  - https://github.com/cardano-foundation/CIPs/pull/1042
```
The bulk of the discussion (including, but not limited to, the change requests that appear in `/files`) generally is linked here.